### DISCUSSION TITLE
C++ Micro Services 3.6.1

### DISCUSSION THREAD

**Discussion by jeffdiclemente** [CppMicroServices/CppMicroServices#764] [Announcements]
### This is a C++14 compliant release of C++ Micro Services.

This version contains all changes from v3.7.2 except the ones introducing C++17 functionality and removing abseil. Additionally there have been 3rd-party dependency updates and some compiler warning fixes not in v3.7.2. Same as for v3.7.2, this list does not include every change since v3.6.0, but only the relevant changes.

<hr /><em>This discussion was created from the release <a href='https://github.com/CppMicroServices/CppMicroServices/releases/tag/v3.6.1'>C++ Micro Services 3.6.1</a>.</em>

**Comment by arunbuddha** [comment]
Hi All,
When I compile 3.6.1 or 3.7.4, I get error while cmake come across third_party/boost/nowide. The CMakeLists.txt was missing and the directory was empty. I even downloaded boost's nowide source and copied there in that directory. 

After that the CMake passed over that state and got the below error.

CMake Error at tools/rc/CMakeLists.txt:13 (add_executable):
  Target "usResourceCompiler" links to target "nowide::nowide" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?

Please explain me the mistake I am doing here.

**Reply by jeffdiclemente** [reply]
did you clone the repo using "--recurse-submodules" command line options (e.g. git clone --recurse-submodules ....)? we have a number of submodules in the repo and they need to be synced either during cloning of the repo or afterwards.

**Reply by arunbuddha** [reply]
Sorry, I did not use that option. After I did that, it compiled well. Very thanks.
